tests/integration/test_enrollment/conftest.py

```diff
@@ -0,0 +1,307 @@
+import os
```

Comment:
The module is not pep8 compliant. 
![3](https://user-images.githubusercontent.com/13010397/131728255-36e3df15-6b90-417e-9c35-c459696b3de1.png)

Reply:
Fixed